test/e2e/atlas/streams_test.go

```diff
@@ -300,6 +303,61 @@ func TestStreams(t *testing.T) {
 		a.Equal("SSL", connection.Security.GetProtocol())
 	})
 
+	t.Run("Creating a streams connection with atlas cluster", func(t *testing.T) {
+		clusterName := "Cluster1"
+
+		cmd := exec.Command(cliPath,
+			clustersEntity,
+			"create",
+			clusterName,
+			"--backup",
+			"--tier", tierM10,
+			"--region=US_EAST_1",
+			"--provider", e2eClusterProvider,
+			"--mdbVersion", mdbVersion,
+			"--projectId",
+			g.projectID,
+			"-o=json")
```

Comment:
any reason not to use the create cluster from the generator?

Reply:
Thanks for the suggestion - I've changed this to use use the generateCluster and the file is being generated with a template. [e5050e8](https://github.com/mongodb/mongodb-atlas-cli/pull/2600/commits/e5050e806369a90294a9ef139202addb89436f81)